src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        int index = new Random().nextInt(100);
+        return new Ball(new ColorSupplier().getRandomColor(), index).toString();
+    }
```

Comment:
```suggestion
    public Ball getRandomBall() {
        int index = random.nextInt(MAX_NUMBER);
        return new Ball(colorSupplier.getRandomColor(), index));
    }
```
lets make Random and ColorSupplier class level variables - to avoid creating a new instance each time this method is invoked
this method should return Ball